core/server/models/plugins/filter.js

```diff
@@ -192,20 +192,35 @@ filter = function filter(Bookshelf) {
 
         // @TODO: remove me when NQL supports counting, it only serves one use case for now (content v2 serves no authors without posts)
         applyCount(options) {
-            if (options.filter && options.filter.match(/authors\.posts:>0/)) {
-                if (options.filter === 'authors.posts:>0') {
-                    delete options.filter;
-                } else {
-                    options.filter = options.filter.replace('+authors.posts:>0', '');
+            if (options.filter) {
+                const filterMatch = options.filter.match(/(authors|tags)\.posts:>0/i);
```

Comment:
Experimented a bit with making tags count filter a bit more generic to avoid code duplication. Just a suggested implementation here def. needs more discussion tomorrow :relaxed:

Reply:
Could be simplified by using `qb._single.table`.